docs/README.md

```diff
@@ -6,27 +6,160 @@ features:
 - title: Your Components
   details: Supports your own form-input components
 - title: Validations!
-  details: Use Vuelidate-compatible schema for your validation rules
+  details: Use Vuelidate-compatible schema for your validation rules (WIP)
 - title: Zero Dependencies
-  details: VuePress.
+  details: dependencies <= 0
 footer: MIT Licensed | Copyright © 2019-present Damian Dulisz
 ---
 
-## Example
+## Getting Started
 
-<SimpleExample></SimpleExample>
+FormVueLatte is a zero dependency library that allows you to generate schema-driven forms with extreme ease.
 
-## With Array Schema
+The schema that you use for your form can be as flexible as you need it to be, it can be modified at run-time with an expected reactive result, and can even be fetched directly from you backend's API.
+
+### Installation
+
+NOTE: This whole section is WIP.
+
+To add FormVueLatte to your project, start by installing the package through your favorite package manager.
+
+```bash
+yarn add formvuelatte
+```
+
+```bash
+npm install formvuelatte
+```
+
+Now that you have the package in your project, `import` it to your component.
+
+```js
+import { SchemaForm } from 'formvuelatte'
+```
+
+Finally, add it to your component and pass it your **schema** through the `:schema` property.
+
+```js
+<template>
+    <SchemaForm :schema="mySchema" />
+</template>
+```
+
+The `SchemaForm` will `$emit` **input** events when your components update. This means that you are able to either use `v-model` on it, or manually capture the `@input` event with a method of your own, as well as inject the `:value` property. Either or.
+
+## Component Requirements and the FormMixin
+In order for FormVueLatte's `SchemaForm` to understand your components, they need to implement the `FormMixin.js` that FormVueLatte provides. In its simplest form, make sure that when your `form element` updates, you call the `update` function with the new value.
```

Comment:
> In its simplest form, make sure that when your `form element` updates, you call the `update` function with the new value.

I think I would rephrase it to something along the lines:

> In it’s simplest form, make sure that when your `form element` wants to make a change to the `value`, it calls the `update` method with the new value.

Reply:
Done and pushed